### ISSUE TITLE
[DEV] Conversation

### ISSUE THREAD

**Issue by TheKillerTacos** [PocketPanel/PocketPanel#3]
RULES:
* Only Devs
* Swearing allowed, no Shit given
* NOT PORN PLS
* And do not offend Devs, plz no.

**Comment by AndreTheGamer** [comment]
I think we should have 2 things to make it more unique.

1) a plugin, which has to be added to the server. The plugins then gets information thats needed by step 2.

2) a web panel /site: Where the admin or even players can view stats about the server, the minimap if they have the plan ...ect). 

Please note that those are just suggestions. You are welcome to share your ideas :)

**Comment by TheKillerTacos** [comment]
yes that was the main idea, but i was going to make novapanel but i abandoned the project to create now called PocketPanel Advance, but yea, ill just create a free website by the moment, like
panel.pocketpanel.tk, but we should change PocketPanel (Organization name) to PocketProjects or something,

**Comment by AndreTheGamer** [comment]
Done i renamed it! and also what were the features of NovaPanel?

**Comment by AndreTheGamer** [comment]
I've renamed it and create another repo on the same name and rediect it here! I's created to avoid impersonator -.-

Anyway as i was saying, what's the features of NovaPanel? Maybe we can try to add it on PocketPanel and who knows... it might be a huge project :P

**Comment by TheKillerTacos** [comment]
The same as PocketPanel, just that it was online and used PocketStats to monitor your server, and mostly manage it Via RCON, but i dont know much about Rcon, so i dropped it out,

Also i thought

PocketPanel Free - Online Panel with Basic Features like PocketStats,
PocketPanel Pro - Manage Server Online and Client, More features, Not free $3.99 Monthly maybe?
PocketPanel Advance - Manage Multiple Server at the time (using Tabs or Sections) Mini-Map and More features.

**Comment by AndreTheGamer** [comment]
Yea, how about the ability to start/stop your server? install plugins/ftp ..ect) Similar to Multicraft but more advanced/unique for pocket edition user(S) ?

**Comment by TheKillerTacos** [comment]
UPDATE:

PocketPanel Guest - Online Panel With PocketPanel, Using GUPI (Generated Unique Password ID)*
PocketPanel - Manage Server with Online Panel, Using "UserKey 1.0"** and GUPI For better support***
PocketPanel Advance - Multiple Server Management, Minimap, Google Map Support, and other, not free, im thinking about the price

GUPI: Is a Generated User Password ID, which is generated via the server, Won't say how is generated, to avoid Hackers, the GUPI, is used when the User (Client) tries to log in to the server, ej, using Guest and connecting with the IP from the server, the server will send back that it need a password, which is GUPI, that can be seen on a File Generated with another plugin given, ill make a wiki page about this in the Plugin Wiki, ill tell you more later on there

**Comment by TheKillerTacos** [comment]
Yes, mostly like multicraft, But you can manage your server with less difficulty, ill make a chart of the login/account structure.

**Comment by TheKillerTacos** [comment]
but first let me finish the logo :L

**Comment by TheKillerTacos** [comment]
![logo](https://cloud.githubusercontent.com/assets/14338036/16680595/568cbf62-44b5-11e6-9251-f5f7569cf692.png)

**Comment by TheKillerTacos** [comment]
![explanation](https://cloud.githubusercontent.com/assets/14338036/16680909/810e00be-44b7-11e6-85bb-a606090448aa.png)
at 7 - The User/Client can send any commands available from server and from the Panel,

also when doing guest login, it creates a Temporal User key (TUK) for the IP from the client, si TUK replases Userkey in this case, just that less features, like sending stop command, and turn on/shut down server.

Also everything that every user does, it will be logged to avoid hackers and to find any suspicious actions, Logs will be deleted every month, so, ill be working on server side (Auth and Connections)

**Comment by AndreTheGamer** [comment]
Yea, sounds good.

**Comment by AndreTheGamer** [comment]
I was thinking, to make this more better. They can install the web-server. Part on the webhosting or even dedicated server hosting then install it using the www.example.com/install.php ..ect). Like instead of us hosting it on the site. They can just visit PocketPanel site and view features of it and download the free version? or even maybe download the premium plan then they are grant a license key

**Comment by TheKillerTacos** [comment]
yes, User key Main purpose is to save all user Data to that Key, and is safer than using Username and Password, And allows to use the Same Username, since the login be like: Username and User Key,
I cant tell how it works, later we can chat on an IRC or something, To avoid people using our method and avoid crackers and Hackers

**Comment by TheKillerTacos** [comment]
also GUPI will be renamed to GSPI, since its actually "Generated **Server** Password ID", becuz the server has that as pasword.

**Comment by SkygridRex** [comment]
Use another css framework like materialize
https://github.com/Dogfalo/materialize

**Comment by AndreTheGamer** [comment]
Yea

**Comment by AndreTheGamer** [comment]
Is here fine doing the whole login/register system? while other devs work on other things?

**Comment by TheKillerTacos** [comment]
Im on Auth side, why not go to[ this](http://webchat.freenode.net/?channels=/PocketPanel) irc and tell you how it works?

**Comment by SkygridRex** [comment]
![captura](https://cloud.githubusercontent.com/assets/13483754/16710591/774b4562-45f9-11e6-8180-9e74c21e6d0f.PNG)
i click the login button, i guess login.php is missing

**Comment by TheKillerTacos** [comment]
im going to work on that

**Comment by TheKillerTacos** [comment]
Im actually sending the website to a Temporal webserver

**Comment by AndreTheGamer** [comment]
irc seems empty. lol no body is even there! :yum:

**Comment by TheKillerTacos** [comment]
well
i was there, let me try connecting via that link

**Comment by AndreTheGamer** [comment]
I think we need a glitter chat instead. I hate IRC!! :P

**Comment by SkygridRex** [comment]
gitter* :yum:

**Comment by AndreTheGamer** [comment]
ye

**Comment by SkygridRex** [comment]
Devs: Thanks Diego :yum:
Me: :yum:

**Comment by TheKillerTacos** [comment]
ok

**Comment by TheKillerTacos** [comment]
well, Why not make index.php the landing page, ill make the **CONTROL PANEL** and make better backgrounds, (CSS Gradients) and some CSS Animations

**Comment by AndreTheGamer** [comment]
-.-

**Comment by TheKillerTacos** [comment]
:L, Bueno, Porque no hacer index.php la pagina principal, yo hare el **PANEL DE CONTROL** y hare mejores fondos (Gradientes en CSS) y unas animaciones CSS

**Comment by SkygridRex** [comment]
Yo podría hacer el login.php (cuando tenga algo de tiempo)

**Comment by TheKillerTacos** [comment]
I'll be away a couple of weeks since I'm going to a Field trip. So.... see ya later! Maybe soon...
#StrangerThings
